complete/pom.xml

```diff
@@ -27,6 +27,14 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
```

Comment:
Because of the parent you don't need all this. You just need a property java.version=1.7.